ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-CP-nonprod.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "requestForReviewCommentsClaimant",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "CITIZEN-CLAIMANT-PROFILE",
+          "CITIZEN-DEFENDANT-PROFILE",
+          "citizen-profile"
+        ],
+        "CRUD": "CRU"
+      }
+    ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "requestForReviewCommentsDefendant",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "CITIZEN-CLAIMANT-PROFILE",
```

Comment:
Claimant should probably just have read access, right?

Reply:
yes, it is more precise, I will modify it